AndelaOSP/Andela-Socials
# #165853787 Notify attendees on slack when host updates an event

#### What Does This PR Do?

- This PR implements slack notifications for all attendees for an event when the host changes the venue or date for the event
#### Description Of Task To Be Completed
- [x] Notify attendees on slack when the venue, date/time for an event changes
- [x] Notify attendees on slack when an event is canceled
#### Any Background Context You Want To Provide?

- N/A
#### How can this be manually tested?

- Create an event
- Add yourself to the attendee list, you could do that by manually adding the info to the api_attend table
- Update an event
- You should receive a slack notification

#### What are the relevant pivotal tracker stories?

[#165853787](https://www.pivotaltracker.com/story/show/165853787)

#### Screenshot(s)
<img width="638" alt="Screen Shot 2019-05-14 at 5 07 35 AM" src="https://user-images.githubusercontent.com/9039613/57671267-ea0d4700-760a-11e9-9789-38093a01297b.png">
